routes/api/contacts.js

```diff
@@ -1,25 +1,101 @@
-const express = require('express')
+const express = require("express");
+const Joi = require("joi");
+const {
+  listContacts,
+  getContactById,
+  addContact,
+  removeContact,
+  updateContact,
+} = require("../../models/contacts");
 
-const router = express.Router()
+const router = express.Router();
 
-router.get('/', async (req, res, next) => {
-  res.json({ message: 'template message' })
-})
+// Validation schema for POST and PUT
+const contactSchema = Joi.object({
```

Comment:
To mozna dać do jakiegoś pliku np validation.js i tam trzymać wszystkie validacje na przyszłość.